pkg/operator/controller/ingress/deployment.go

```diff
@@ -80,6 +80,8 @@ const (
 
 	RouterHAProxyConfigManager = "ROUTER_HAPROXY_CONFIG_MANAGER"
 
+	RouterHAProxyContstats = "ROUTER_HAPROXY_CONTSTATS"
```

Comment:
https://github.com/openshift/router/pull/449 adds `ROUTER_CONTSTATS`, and here you have `ROUTER_HAPROXY_CONTSTATS`.

Reply:
@Miciah yes, thank you for spotting. I'd failed to push a change to my upstream branch for https://github.com/openshift/router/pull/449.  It is done now.